worldclockwidget/build.gradle

```diff
@@ -109,7 +109,7 @@ dependencies{
     compile 'com.actionbarsherlock:actionbarsherlock:4.4.0@aar'
     compile 'com.android.support:support-v4:19.1.0'
     compile 'com.google.code.gson:gson:2.3.1'
-    compile 'net.danlew:android.joda:2.9.4.1'
+    compile group: 'net.time4j', name: 'time4j-android', version: '3.41-2018d'
```

Comment:
This version should be updated to 3.44-2018e so at least two warnings or more of the travis build would disappear.

Reply:
The build complains mainly about two unresolved references to library class members. This problem is solved by the newer version 3.44-2018e. About Proguard: There is already a special consumer-proguard-file in the library. For example, let's look at the build note

Note: net.time4j.base.ResourceLoader calls '(net.time4j.base.ResourceLoader)Class.forName(variable).newInstance()'

This should not be a problem because the delivered proguard-file already keeps the class `net.time4j.android.spi.AndroidResourceLoader`, see also the [file content](https://github.com/MenoData/Time4A/blob/master/time4j-android/consumer-proguard-rules.pro).